packages/fiori-elements-writer/test/feop.test.ts

```diff
@@ -56,6 +57,8 @@ describe(`Fiori Elements template: ${TEST_NAME}`, () => {
             } else {
                 resolve(true);
             }
+        }).then(async () => {
+            await projectChecks(testPath, config, debug?.debugFull);
```

Comment:
Shouldnt wrap each call to `projectChecks` like so for clarity :
```suggestion
    if (debug.debugFull) {
            await projectChecks(testPath, config);
     }
```

Reply:
I had it like that first locally and it seemed like more duplication than to do the check inside projectChecks() once.

Reply:
I dont think its usual to encapsulate the entire function internals behind a passed options. Seem like a smell to me and decoupling and clarity would trump duplication here?